cpp/roaring64map.hh

```diff
@@ -1092,6 +1093,23 @@ public:
         return result;
     }
 
+    /**
+     * Returns the number of integers that are smaller or equal to x.
+     */
+    int128_t getIndex(uint64_t x) const {
+        uint64_t index = 0;
+        auto roaring_destination = roarings.find(highBytes(x));
+        if (roaring_destination != roarings.cend()) {
+            for (auto roaring_iter = roarings.cbegin();
+                 roaring_iter != roaring_destination; ++roaring_iter) {
+                index += roaring_iter->second.cardinality();
+            }
+            index += roaring_destination->second.getIndex(lowBytes(x));
```

Comment:
@bryceustc  it is a bug to add `-1` to `index`

Reply:
and the type of `index` should be `int128_t`

Reply:
Thanks for your review, the bug and comments have been fixed in the latest commit.